Dockerfile

```diff
@@ -1,9 +1,8 @@
 FROM golang:1.13 as secrets-provider-builder
-MAINTAINER CyberArk Software, Ltd.
+MAINTAINER CyberArk Software Ltd.
 
 ENV GOOS=linux \
-    GOARCH=amd64 \
-    CGO_ENABLED=0
```

Comment:
why was this removed? it should still be here (although it's the default value, it's best to have it for explicitness)